streams/src/main/java/org/apache/kafka/streams/kstream/Window.java

```diff
@@ -22,8 +22,8 @@
  */
 public abstract class Window {
 
-    private long start;
-    private long end;
+    protected long start;
```

Comment:
We can. I still have #2337 open that needs to get rebased after this got merged -- will fix it there if ok.